packages/web/components/swap-tool/index.tsx

```diff
@@ -528,14 +544,14 @@ export const SwapTool: FunctionComponent<{
                       {t("swap.available")}
                     </span>
                     <span className="caption ml-1.5 text-sm text-wosmongton-300 md:text-xs">
-                      {queries.queryBalances
-                        .getQueryBech32Address(account?.address ?? "")
-                        .getBalanceFromCurrency(tradeTokenInConfig.sendCurrency)
-                        .trim(true)
-                        .hideDenom(true)
-                        .maxDecimals(8)
-                        .toString()}{" "}
-                      {tradeTokenInConfig.sendCurrency.coinDenom}
+                      {formatPretty(
+                        queries.queryBalances
+                          .getQueryBech32Address(account?.address ?? "")
+                          .getBalanceFromCurrency(
+                            tradeTokenInConfig.sendCurrency
+                          ),
+                        { maxDecimals: 2 }
+                      )}
```

Comment:
I personally liked the previous decimal precision better. It gave me a better idea of the balance available. What if we increase the `maxDecimals` to `4`?

Reply:
How about this approach: if we have a number that is less than `1`, we display it with `6` decimals; but if the number is 1 or more, we show it with just `2` decimlas?